### PR TITLE
Added CI workflow with a job to format the code on the PR

### PR DESCRIPTION
**Pull request by hsein-bitar**
## Asana
https://app.asana.com/0/1202852195727075/1205992270273982/f

## Context
Added CI workflow with a job to format the code on the PR

### CODE REVIEW

**Comment by hsein-bitar** [comment]
hello, this should be ready.
The PR includes lots of file changes introduced by the CI pipeline formatting many files.
Outside of the formatting, [the main changes can be viewed using this link, with some JS formatting to be ignored](https://github.com/nsquared-team/draw-attention/pull/50/files/3ca8093620c511bc07ba580895fbcefe7f697c1e):
- We introduced ci.yml workflow, which runs only on pull requests, and attempts to format and catch any code errors
- The workflow relies on some configuration files, so I also added `.prettierignore`, `.eslintrc.js`, and `composer files` 


## Testing
I tested a Draw Attention element that I had previously setup:
- confirmed it can still be viewed as expected ✅
- confirmed it still has the settings and can be edited as expected ✅
- added new hotspots to it and confirmed it can be updated with no issues ✅

I also tested replacing the plugin with a new build, and confirmed it works ✅

File: public/assets/js/leaflet.js
Comment (NatalieMac) [inline public/assets/js/leaflet.js:1]: It looks like we're attempting to un-minify this file. Is there a way to just exclude this file?

**Review by NatalieMac — CHANGES_REQUESTED** [review]

**Comment by NatalieMac** [comment]
I looped in Cyndy to do some QA just in case.

File: public/assets/js/leaflet.js
Comment (hsein-bitar) [reply public/assets/js/leaflet.js:1]: Oh sure, thanks for catching this. I had excluded all files matching `min.*` but missed this one. Will add it now

**Review by tylerdigital — APPROVED** [review]
@cynhu92 @gnarza please fully try out this version of Draw Attention to make sure we haven't accidentally broken anything. This shouldn't affect anything, but we're modifying a lot of files throughout the codebase

**Review by cynhu92 — CHANGES_REQUESTED** [review]
I was testing this, but it seems to be only DA Free, right? Do we have a Pro version to test with?

**Review by cynhu92 — APPROVED** [review]
Was doing my testing here: https://pr50-drawattention-cyn.sandbox.ssa.rocks/test/

Tested:
- Creating image ✅
- Deleting image ✅ 
- Replacing image ✅
- Creating hotspots ✅
- Deleting hotspots ✅
- Editing hotspots ✅
- Using polygon, rectangle, circle hotspots ✅
- Adding shortcodes and links to hotspots ✅
- Adding videos ✅
- Using Go to Url and More info ✅
- Changing hotspot styles and borders ✅

**Comment by NatalieMac** [comment]
> I was testing this, but it seems to be only DA Free, right? Do we have a Pro version to test with?

Draw Attention free and Pro are two totally separate repositories (unlike SSA where all versions are in one repo), so they have to be updated and tested separately. @cynhu92

**Review by gnarza — APPROVED** [review]
Tested on sandbox:
Adding main image ✅ 
Creating hotspot polygon ✅ 
Creating hotspot square ✅ 
Creating hotspot circle ✅ 
Editing hotspots ✅ 
Zooming and panning ✅ 
Deleting hotspots ✅ 
Show description action ✅ 
Go to URL action ✅ 
Attach Detail Image ✅ 
Highlight Styling ✅ 
More Info Box Styling ✅ 
Apply Color Scheme ✅ 

Tested on fresh local:
Adding main image ✅ 
Creating hotspot polygon ✅ 
Creating hotspot square ✅ 
Creating hotspot circle ✅ 
Editing hotspots ✅ 
Zooming and panning ✅ 
Deleting hotspots ✅ 
Show description action ✅ 
Go to URL action ✅ 
Attach Detail Image ✅ 
Highlight Styling ✅ 
More Info Box Styling ✅ 
Apply Color Scheme ✅

**Pull request merged**